lib/manager/bundler/extract.ts

```diff
@@ -1,6 +1,5 @@
 import { RubyGemsDatasource } from '../../datasource/rubygems';
```

Comment:
I think a PR title like `fix(bundler): handle missing versions for gems correctly` is better, we don't really use the `manager` scope. 😉

Reply:
`manager/bundler` would be best. 😉

Reply:
Sure, I've updated it. :)